content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -422,7 +422,7 @@ fail validation.
 			<td>
 				<p>
 					Containers must drop <code>ALL</code> capabilities, and are only permitted to add back
```

Comment:
This check WILL be OS-specific once that support is added to pods, and will not apply to windows pods, but we should not update this doc until that feature lands (planned for alpha in 1.23)

Reply:
Right. This PR is supposed to merge after the code changes are in place.

Reply:
See https://github.com/kubernetes/enhancements/issues/2802#issuecomment-922257794 for the request for docs.